stable/redis-ha/values.yaml

```diff
@@ -11,6 +11,16 @@ replicas: 3
 ## Custom labels for the redis pod
 labels: {}
 
+## Pods Service
+## ref: https://kubernetes.io/docs/concepts/services-networking/service/#defining-a-service
+service:
+  ## Specifies the Service type. Supports: LoadBalancer. Default (Empty): ClusterIP
+  ##
+  type: LoadBalancer
```

Comment:
Sorry @DandyDeveloper I canceled my `lgtm` comment because I didn't realize this was changed as well. Is there a reason why this is changed from the last default or was this just a remnant of your testing? My worry main concern is that this is a breaking change for upgrades.

Reply:
Because I'm an absolute idiot who did not revert back his testing! Sorry, I'll change it back quickly, I actually made this PR to test our environment requirements for an LB.

Reply:
@ssalaues Fixed. Sorry about the delay (in Japan, timezones).